salt/modules/win_pkg.py

```diff
@@ -913,18 +913,6 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
                 # Version 1.2.3 will apply to packages foo and bar
                 salt '*' pkg.install foo,bar version=1.2.3
 
-        cache_file (str):
-            A single file to copy down for use with the installer. Copied to the
-            same location as the installer. Use this over ``cache_dir`` if there
-            are many files in the directory and you only need a specific file
-            and don't want to cache additional files that may reside in the
-            installer directory. Only applies to files on ``salt://``
```

Comment:
Please move this to `salt/doc/topics/windows/windows-package-manager.rst` right above `cache_dir` (approx line 480)

Reply:
Done